x-pack/solutions/security/plugins/asset_inventory/public/components/app.tsx

```diff
@@ -6,33 +6,26 @@
  */
 import React from 'react';
 import { FormattedMessage, I18nProvider } from '@kbn/i18n-react';
-import { BrowserRouter as Router } from '@kbn/shared-ux-router';
 import { EuiPageTemplate, EuiTitle } from '@elastic/eui';
-import type { CoreStart } from '@kbn/core/public';
 
-interface AssetInventoryAppDeps {
-  basename: string;
-  notifications: CoreStart['notifications'];
-  http: CoreStart['http'];
-}
-
-export const AssetInventoryApp = ({ basename }: AssetInventoryAppDeps) => {
+const AssetInventoryApp = () => {
   return (
-    <Router basename={basename}>
-      <I18nProvider>
-        <>
-          <EuiPageTemplate restrictWidth="1000px">
-            <EuiPageTemplate.Header>
-              <EuiTitle size="l">
-                <h1>
-                  <FormattedMessage id="assetInventory.helloWorldText" defaultMessage="Inventory" />
-                </h1>
-              </EuiTitle>
-            </EuiPageTemplate.Header>
-            <EuiPageTemplate.Section />
-          </EuiPageTemplate>
-        </>
-      </I18nProvider>
-    </Router>
+    <I18nProvider>
+      <>
+        <EuiPageTemplate restrictWidth="1000px">
+          <EuiPageTemplate.Header>
+            <EuiTitle size="l">
+              <h1>
+                <FormattedMessage id="assetInventory.allAssets" defaultMessage="All Assets" />
```

Comment:
We have an existing assets page on the serverless version:
<img width="2555" alt="Screenshot 2024-12-23 at 17 31 17" src="https://github.com/user-attachments/assets/9f34c403-5b1d-469a-b462-ddd052ef25a4" />

I'd recommend making this header more specific to avoid confusion:
<img width="2556" alt="Screenshot 2024-12-23 at 17 17 02" src="https://github.com/user-attachments/assets/46895b9c-86f9-4b1d-a2b0-169b1c9149c4" />

Reply:
Hmm I can bring it up in the next UX weekly meeting. I'm just following this [Figma mockup](https://www.figma.com/design/9zUqAhhglT1EGYG4LOl1X6/Asset-Management?node-id=3068-34003&t=bFygEtfN5kP0ZIhQ-0).

<img width="1456" alt="Screenshot 2024-12-23 at 18 42 38" src="https://github.com/user-attachments/assets/4ed87277-70c3-4355-9822-321cc78749fa" />